sass.gemspec

```diff
@@ -21,19 +21,17 @@ SASS_GEMSPEC = Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.0.0'
 
   spec.add_runtime_dependency 'sass-listen', '~> 4.0.0'
-  
+
   spec.add_development_dependency 'yard', '~> 0.8.7.6'
   spec.add_development_dependency 'redcarpet', '~> 3.3'
   spec.add_development_dependency 'nokogiri', '~> 1.6.0'
   spec.add_development_dependency 'minitest', '>= 5'
 
   readmes = Dir['*'].reject{ |x| x =~ /(^|[^.a-z])[a-z]+/ || x == "TODO" }
   spec.executables = ['sass', 'sass-convert', 'scss']
-  spec.files = Dir['rails/init.rb', 'lib/**/*', 'bin/*', 'test/**/*',
-    'extra/**/*', 'Rakefile', 'init.rb', '.yardopts'] + readmes
```

Comment:
Why are you removing `.yardopts` and `init.rb`?

Reply:
I have reverted .yardopts